example/src/main/kotlin/dev/patrickgold/jetpref/example/AppPrefs.kt

```diff
@@ -18,6 +21,14 @@ class AppPrefs : PreferenceModel("example-app-preferences") {
         key = "theme",
         default = Theme.AUTO,
     )
+    val accentColor = custom<Color>(
+        key = "accent_color",
+        default =  if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S)
+            Color.Unspecified
+        else
+            Color.Red,
+        serializer = ColorPreferenceSerializer
+    )
```

Comment:
A slightly cleaner way to define the pref would be this:

```suggestion
    val accentColor = custom(
        key = "accent_color",
        default = when (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
            true -> Color.Unspecified
            false -> Color.Red
        },
        serializer = ColorPreferenceSerializer,
    )
```